### ISSUE TITLE
UI design

### ISSUE THREAD

**Issue by eisisig** [eisisig/react-isolate#17]
There wasn't any real thought put into the design so It might be a good idea to work on that.

The arrangement of the panels, side-menu, resizable panels etc..

**Comment by jsergiu** [comment]
Would be nice to have a button to toggle the examples panel size: 50% <-> 100%.

**Comment by eisisig** [comment]
Yeah but should we store the "users" settings in localStorage or? So it would be the same the next time or?
